### PR TITLE
feat: US136980 - add LOR import feature to FACE quizzing. 

### PR DESCRIPTION
**Pull request by alexli01**
1. Call a new api to open LOR dialog
2. Call a new api to import selected LOR objects

### CODE REVIEW

File: components/d2l-activity-editor/d2l-activity-quiz-editor/d2l-activity-quiz-add-activity-menu.js [hunk 1]
```diff
@@ -8,6 +8,8 @@ import { RtlMixin } from '@brightspace-ui/core/mixins/rtl-mixin.js';
 import { SkeletonMixin } from '@brightspace-ui/core/components/skeleton/skeleton-mixin.js';
 import { shared as store } from './state/quiz-store.js';
 
+
+
```
Comment (austinkwok1997) [inline components/d2l-activity-editor/d2l-activity-quiz-editor/d2l-activity-quiz-add-activity-menu.js:12]: There are some extra empty lines added.

File: components/d2l-activity-editor/d2l-activity-quiz-editor/d2l-activity-quiz-add-activity-menu.js [hunk 2]
```diff
@@ -94,11 +96,13 @@ class ActivityQuizAddActivityMenu extends ActivityEditorMixin(SkeletonMixin(Loca
...
 			url = new D2L.LP.Web.Http.UrlLocation(`/d2l/lms/question/upload/${type}`);
-		}
-		else {
+		} else if (type.includes('importLOR')) {
+			url = new D2L.LP.Web.Http.UrlLocation(`/d2l/lms/question/openLOR/${type}`);
+		} else {
 			url = new D2L.LP.Web.Http.UrlLocation(`/d2l/lms/question/new/${type}`);
 		}
 
+
```
Comment (austinkwok1997) [inline components/d2l-activity-editor/d2l-activity-quiz-editor/d2l-activity-quiz-add-activity-menu.js:105]: Extra empty line here too

File: components/d2l-activity-editor/d2l-activity-quiz-editor/d2l-activity-quiz-add-activity-menu.js (hunk 1 above)
Comment (alexli01) [reply components/d2l-activity-editor/d2l-activity-quiz-editor/d2l-activity-quiz-add-activity-menu.js]: fixed

File: components/d2l-activity-editor/d2l-activity-quiz-editor/d2l-activity-quiz-add-activity-menu.js (hunk 2 above)
Comment (alexli01) [reply components/d2l-activity-editor/d2l-activity-quiz-editor/d2l-activity-quiz-add-activity-menu.js]: fixed.

**Review by Scy-D2L — APPROVED** [review]

**Pull request merged**